### DISCUSSION TITLE
Does QB support SOCKS5 proxies with authentication ?

### DISCUSSION THREAD

**Discussion by Stas1981ru** [qutebrowser/qutebrowser#5811] [Q&A] [answered]
My system is:Debian 10, Qt version: 5.15.0 , PyQt version: 5.15.0. QB ver 1.13.1
I have a worked socks5 proxy (it worked fine with curl :" `CURL --socks5-hostname xxx:xxx@x.lrcs.ru:8885 https://www.google.com` ").
But when I set this proxy in QB via `:set content.proxy socks5://xxx:xxx@x.lrcs.ru:8885` and go to any url - it say "`ERR_SOCKS_CONNECTION_FAILED`"

Whats wrong ?

**Comment by The-Compiler** [comment, marked as answer]
Looks like Chromium (and thus qutebrowser) doesn't support SOCKS5 authentication, with no plans to add it: [256785 - SOCKS5 authentication support - chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=256785)
